cryptol-remote-api/python/cryptol/cryptoltypes.py

```diff
@@ -185,62 +228,7 @@ def is_plausible_json(val : Any) -> bool:
     return False
 
 class CryptolType:
-    def from_python(self, val : Any) -> JSON:
```

Comment:
It's possible there is not any code that explicitly depends on the methods on the parent class `CryptolType` that are deleted in this PR (`from_python` and `convert`)... but to be safe, could we keep those methods and just have them call the functions that now have all of that logic? (is that mostly just `to_cryptol` for both...?)

Reply:
As per your later suggestion, I put both of these methods back but added errors to both saying they're deprecated.